src/main/resources/validation.properties

```diff
@@ -0,0 +1,5 @@
+NotEmpty=This field is required.
+Size.userForm.username=Please use between 6 and 32 characters.
+Duplicate.userForm.username=Someone already has that username.
+Size.userForm.password=Try one with at least 8 characters.
+Diff.userForm.passwordConfirm=These passwords don't match.
```

Comment:
Are these used? I ask because It appears these strings/ similar ones are hardcoded into the Validator class

Reply:
You're right, these aren't used. I was originally going to use them, but decided against it because, like you said, it's already in the validator.